bindings/pyroot_experimental/PyROOT/python/ROOT/__init__.py

```diff
@@ -58,3 +58,19 @@ def pythonization_impl(fn):
 import sys
 from ._facade import ROOTFacade
 sys.modules[__name__] = ROOTFacade(sys.modules[__name__])
+
+# Set import hook to be able to trigger auto-loading as appropriate
```

Comment:
Great!

Can we perhaps move this to the facade?

I would encapsulate it in a function that is invoked from (to be tried) `_finalSetup`, or alternatively if it needs to be done earlier, from `__init__` in the facade.

Reply:
And in the documentation of the function, I would explain what is the case that this import hook is covering and show an example (e.g. from ROOT.A import B).